.github/workflows/composer-diff.yml

```diff
@@ -0,0 +1,28 @@
+name: Composer Diff
+on:
+  pull_request:
+    paths:
+      - 'composer.lock'
+permissions:
+  contents: write
```

Comment:
```suggestion
```
Don't think this action is committing anything?

Reply:
this was copy pasta from the last several composer-diff workflows. it might have been added with the idea that it's "writing" a comment to the PR but I don't think that's what this permission is for.